app/models/user.rb

```diff
@@ -75,6 +76,10 @@ def name
     "#{first_name} #{last_name}"
   end
 
+  def email
```

Comment:
This method isn't needed - you should be able to access the email attribute without it.